### PR TITLE
README.md: Add build deps install instructions

### PR DESCRIPTION
**Pull request by GordanM**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,15 @@ Maintainer: Joakim Gross <joakim.gross@pelagicore.com>
 * texlive-latex-extra (when building PDF)
 * texlive-latex-recommended (when building PDF)
 
+###  Install build dependencies on Debian
+
+```
+sudo apt-get install python-pip
+pip install sphinxcontrib-seqdiag sphinxcontrib-blockdiag \
+              sphinxcontrib-actdiag sphinxcontrib-actdiag \
+              sphinxcontrib-manpage
```
Comment (zeenix) [inline README.md:26]: is the pip really needed here? I mean if all packages are available on debian, best just have 1 command?

**Review by zeenix — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (GordanM) [reply README.md:26]: I think these packages are available only through pip. 

I tried `ap-get`-ing them and it didn't work. Maybe @breakreturn knows the details.

Comment (DunderRoffe) [reply README.md:26]: Yes they are available through apt in debian, at least in debian 9. However, we still have to use pip in other places, for instance vagrant-cookbook, to install packages that are not available through apt so in the interest of consistency we should use pip to install python packages even in this case. Furthermore, pip will provide you with a more up to date version.

**Review by rpannek — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (zeenix) [reply README.md:26]: hmm.. ok. I just thought one command instead of two separate ones, would be simpler.

**Review by zeenix — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (breakreturn) [reply README.md:26]: @GordanM No I don't know any details. I can only remember using pip before for these packages. I have no real opinion here, whatever instructions that would work for most users is the best I'd say.

**Review by breakreturn — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (GordanM) [reply README.md:26]: Thanks for the comments!

**Pull request merged**